test/unit/specs/store/Slide.spec.js

```diff
@@ -0,0 +1,1136 @@
+
+        assert(state.currentSlide.meta.timeout, '40')
+        assert(state.isCurrentSlideDirty, true)
+      })
+
+      it('SET_META_REPEATABLE', () => {
+        assert(state.currentSlide.meta.repeatable, null)
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_META_REPEATABLE](state, false)
+
+        assert(state.currentSlide.meta.repeatable, false)
+        assert(state.isCurrentSlideDirty, true)
+      })
+
+      it('SET_META_STARTDATE', () => {
+        assert(state.currentSlide.meta.startDate, null)
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_META_STARTDATE](state, '2018-02-16')
+
+        assert(state.currentSlide.meta.startDate, '2018-02-16')
+        assert(state.isCurrentSlideDirty, true)
+      })
+
+      it('SET_META_ENDDATE', () => {
+        assert(state.currentSlide.meta.endDate, null)
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_META_ENDDATE](state, '2018-02-16')
+
+        assert(state.currentSlide.meta.endDate, '2018-02-16')
+        assert(state.isCurrentSlideDirty, true)
+      })
+    })
+  })
+
+  // TEST GETTERS
+  describe('getters', () => {
+    describe('title', () => {
+      let state
+
+      beforeEach(() => {
+        state = refreshState()
+      })
+
+      it('SET_TITLE', () => {
+        const titleStateBeforeMutation = {
+          content: '',
+          fontColor: null,
+          fontSize: null,
+          fontStyle: null,
+          fontWeight: null
+        }
+
+        const titleStateAfterMutation = {
+          content: 'Slide1',
+          fontColor: 'Red',
+          fontSize: 'Large',
+          fontStyle: 'Normal',
+          fontWeight: 'Bold'
+        }
+        assert(state.currentSlide.title, titleStateBeforeMutation)
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_TITLE](state, {
+          content: 'Slide1',
+          fontColor: 'Red',
+          fontSize: 'Large',
+          fontStyle: 'Normal',
+          fontWeight: 'Bold'
+        })
+
+        assert(slide.getters.getCurrentSlideTitle(state), titleStateAfterMutation)
+        assert(state.isCurrentSlideDirty, true)
+      })
+
+      it('SET_TITLE_CONTENT', () => {
+        assert(state.currentSlide.title.content, '')
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_TITLE_CONTENT](state, 'title')
+
+        assert(slide.getters.getCurrentSlideTitleContent(state), 'title')
+        assert(state.isCurrentSlideDirty, true)
+      })
+
+      it('SET_TITLE_FONT_COLOR', () => {
+        assert(state.currentSlide.title.fontColor, null)
+        assert(state.isCurrentSlideDirty, false)
+
+        slide.mutations[CURRENT_SLIDE.SET_TITLE_FONT_COLOR](state, 'Red')
+
+        assert(slide.getters.getCurrentSlideTitleFontColor(state), 'Red')
+        assert(state.isCurrentSlideDirty, true)
```

Comment:
I think this should have been state.**getters**.isCurrentSlideDirty

Reply:
It should, the reason it is not is I forgot to make a getter for isCurrentSlideDirty.
Good catch, will fix.

Reply:
The missing getter has now been added. ( getIsCurrentSlideDirty)
add has been added to the tests where needed.